src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java

```diff
@@ -276,20 +276,20 @@ public void setPartnerName(String partnerName) {
 		this.partnerName = partnerName;
 	}
 
-	public String getFetosenseMotherID() {
+	public String getFoetalMonitorMotherID() {
 		return cmMotherId;
 	}
 
-	public void setFetosenseMotherID(String fetosenseMotherID) {
-		this.cmMotherId = fetosenseMotherID;
+	public void setFoetalMonitorMotherID(String foetalMonitorMotherID) {
+		this.cmMotherId = foetalMonitorMotherID;
 	}
 
-	public String getFetosensePartnerID() {
+	public String getFoetalMonitorPartnerID() {
 		return partnerId;
 	}
 
-	public void setFetosensePartnerID(String fetosensePartnerID) {
-		this.partnerId = fetosensePartnerID;
+	public void setFoetalMonitorPartnerID(String foetalMonitorPartnerID) {
+		this.partnerId = foetalMonitorPartnerID;
```

Comment:
@swetatech24 @Anjana2310 Can we rename foetalMonitorPartnerID to just partnerId?